Modules/vc-module-store/VirtoCommerce.StoreModule.Core/Services/IStoreService.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading.Tasks;
+using VirtoCommerce.StoreModule.Core.Model;
+using VirtoCommerce.StoreModule.Core.Model.Search;
+
+namespace VirtoCommerce.StoreModule.Core.Services
+{
+	public interface IStoreService
+	{
+        Task<StoreSearchResult> SearchStoresAsync(StoreSearchCriteria criteria);
```

Comment:
Move Search methods to separate service, need to split CRUD and Search operations